Documentation/Input/Pointers.md

```diff
@@ -1,39 +1,239 @@
+
+The default MRTK Pointer Profile configuration includes the following pointer classes and associated prefabs out-of-box. The list of pointers available to the system at runtime is defined under *Pointer Options* in the Pointer profile. Developers can utilize this list to reconfigure existing Pointers, add new Pointers, or delete one.
+
+Each pointer entry pointer is defined by the following set of data:
+
+- *Controller Type* - The set of controllers that a pointer is valid for.
+    * For example, the *PokePointer* is responsible for "poking" objects with a finger, and is, by default marked as only supporting the articulated hand controller type. Pointers are only instantiated when a controller becomes available and in particular the *Controller Type* defines what controllers this pointer prefab can be created with.
+
+- *Handedness* - allows for a pointer to only being instantiated for a specific hand (left/right)
+
+> [!NOTE]
+> Setting the *Handedness* property of a Pointer entry to *None* will effectively disable it from the system as an alternative to removing that Pointer from the list.
+
+- *Pointer Prefab* - This prefab asset will be instantiated when a controller matching the specified controller type and handedness starts being tracked.
+
+It is possible to have multiple pointers associated with a controller. For example, in the [default HoloLens 2 profile](https://github.com/microsoft/MixedRealityToolkit-Unity/blob/mrtk_development/Assets/MixedRealityToolkit.SDK/Profiles/HoloLens2/DefaultHoloLens2InputSystemProfile.asset)
+the articulated hand controller is associated with the *PokePointer*, *GrabPointer*, and the *DefaultControllerPointer* (i.e hand rays).
+
+> [!NOTE]
+> MRTK provides a set of pointer prefabs in *Assets/MixedRealityToolkit.SDK/Features/UX/Prefabs/Pointers*. A new custom prefab can be built as long as it contains one of the pointer scripts in *Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Pointers* or any other script implementing [`IMixedRealityPointer`](xref:Microsoft.MixedReality.Toolkit.Input.IMixedRealityPointer).
+
+![Pointer Options Profile Example](../Images/Input/Pointers/PointerOptionsProfile.png)
+
+### Default pointer classes
+
+The following classes are the out-of-box MRTK pointers available and defined in the default *MRTK Pointer Profile* outlined above. Each pointer prefab provided under *Assets/MixedRealityToolkit.SDK/Features/UX/Prefabs/Pointers* contains one of these pointer components attached.
+
+![MRTK Default Pointers](../Images/Input/Pointers/MRTK_Pointers.png)
+
+
+#### Far pointers
+
+##### [`LinePointer`](xref:Microsoft.MixedReality.Toolkit.Input.MousePointer)
+
+ A base pointer class, draws lines from the source of the input (i.e. the controller) in the pointer direction. This is also used by other things like the Shell Hand Ray Pointer and the teleport pointers (which also draw lines to indicate where teleportation will end up at).
```

Comment:
It might be worth adding a note that this pointer itself isn't directly instantiated, and is only used through subclassing of the shell hand ray pointer and teleport pointers.

The reason I mention this is because the next line contains additional content (i.e. rotation stuff) which makes me think that actually this is more useful itself than it is (i.e. DefaultControllerPointer isn't mentioned here, which is actually the main one that is actually referenced in the profile.)

Reply:
Gah, and it's totally wrong reference

Reply:
Also DefaultControllerPointer is only the prefab which is mentioned earlier on and is mentioned with ShellHandRayPointer